.github/workflows/coderabbit-review.yml

```
@@ -0,0 +1,157 @@
+        required: true
+        type: string
+
+env:
+  CODERABBIT_CONFIG_PATH: '.github/coderabbit.yaml'
+
+jobs:
+  coderabbit-review:
+    name: 🔍 AI Code Review
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request' || github.event_name == 'workflow_dispatch'
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)